### PR TITLE
feat(processes): adjust retriggers

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* add missing endpoints to retrigger failed steps
* add retrigger steps and endpoints for process steps which miss a retrigger step

## Why

There are retrigger steps without an endpoint to retrigger them and some processes are missing a retrigger step.

## Issue

Refs: #912

## CD PR

https://github.com/eclipse-tractusx/portal-assets/pull/386

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by evegufy — COMMENTED** [review]
Did you notice the 4 new issues by sonarcloud?
Can they, or some of them, be addressed?

**Comment by Phil91** [comment]
> Did you notice the 4 new issues by sonarcloud? Can they, or some of them, be addressed?

@evegufy yes, did saw them and had a look at them, those are all because of the migration. I wouldn't address them. But I mark them as won't do on sonar with an appropriate comment

File: src/processes/Processes.Library/RetriggerExtensions.cs
Comment (evegufy) [inline src/processes/Processes.Library/RetriggerExtensions.cs:1]: please add file header

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/processes/Processes.Library/RetriggerExtensions.cs
Comment (ntruchsess) [reply src/processes/Processes.Library/RetriggerExtensions.cs:1]: done

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**